Doc/library/winreg.rst

```diff
@@ -146,7 +146,9 @@ This module offers the following functions:
 
    .. note::
       The :func:`DeleteKeyEx` function is implemented with the RegDeleteKeyEx
-      Windows API function, which is specific to 64-bit versions of Windows.
+      Windows API function, which is intended to be used for 64-bit Windows.
+      On 32-bit systems, it is strongly recommended to use :func:`DeleteKey()`
+      instead.
```

Comment:
I did some more research, and it seems that there's no distinction on any modern Windows OS. `RegDeleteKey` just calls `RegDeleteKeyEx` directly and passes in zero for the extra parameter, so there isn't really any need for this note at all.

Reply:
Got it, I moved the note completely.